packages/atlas/src/providers/user/user.provider.tsx

```diff
@@ -67,7 +68,7 @@ export const UserProvider: FC<PropsWithChildren> = ({ children }) => {
       }
 
       try {
-        const initializedAccounts = await initSignerWallet(walletName)
+        const initializedAccounts = await retryPromise(() => initSignerWallet(walletName), 500, 2000)
```

Comment:
![image](https://user-images.githubusercontent.com/51168865/214308758-d6f1d377-8f51-46d0-97e5-54acb63e97c9.png)
I'm getting an error, when I want to authorize the app, the error won't disappear until I refresh the site. This didn't happen on `dev`

To reproduce: 
1. Run `yarn atlas:dev` on a different port for example `yarn atlas:dev --port 4333`. This will be helpful to trigger the authorization popup
2. Click connect wallet 
3. Select Polkadot extension and click Select Wallet
4. The authorization popup should show
5. Wait 2 seconds, and you'll notice the error above
6. Once you authorize error stays and won't disappear until you refresh the site.

Reply:
Actually, I just found out you can trigger the authorization popup easier.
1. Disconnect the wallet
2. Click the Polkadot extension icon in your browser
3. Click the cog icon to open the settings, then click manage website access
4. Find localhost:3000 or 127.0.0.1:3000 and click the trash button
5. Connect the wallet again